interpreter/syntax/ast.ml

```diff
@@ -287,10 +287,10 @@ let export_type (m : module_) (ex : export) : extern_type =
   | GlobalExport x ->
     let gts = globals its @ List.map (fun g -> g.it.gtype) m.it.globals in
     ExternGlobalType (nth gts x.it)
-  | EventExport x ->
-    let ets =
-      events its @ List.map (fun (e : event) -> func_type_for m e.it.etype) m.it.events
-    in ExternEventType (nth ets x.it)
+  | TagExport x ->
+    let tts =
+      tags its @ List.map (fun t -> func_type_for m t.it.tgtype) m.it.tags
```

Comment:
Yeah I just made this change to be more consistent with the other cases in that function (which don't have the annotation), but it should work either way because the type is inferred.